internal/appliance/defaults.go

```diff
@@ -27,26 +26,39 @@ func newDefaultConfig() Sourcegraph {
 			},
 			RepoUpdater: RepoUpdaterSpec{
 				StandardConfig: config.StandardConfig{
-					PrometheusPort: ptr.To(6060),
+					PrometheusPort: pointers.Ptr(6060),
 				},
 			},
 			StorageClass: StorageClassSpec{
 				Name: "sourcegraph",
 			},
 			Symbols: SymbolsSpec{
 				StandardConfig: config.StandardConfig{
-					PrometheusPort: ptr.To(6060),
+					PrometheusPort: pointers.Ptr(6060),
 				},
 				Replicas:    1,
 				StorageSize: "12Gi",
 			},
 			GitServer: GitServerSpec{
 				StandardConfig: config.StandardConfig{
-					PrometheusPort: ptr.To(6060),
+					PrometheusPort: pointers.Ptr(6060),
 				},
 				Replicas:    1,
 				StorageSize: "200Gi",
 			},
+			PGSQL: PGSQLSpec{
+				StandardConfig: config.StandardConfig{
+					PrometheusPort: pointers.Ptr(6060),
```

Comment:
```suggestion
					PrometheusPort: pointers.Ptr(9187),
```

See compare-helm diff comment

Reply:
I noticed this, but I don't know if there is a good reason it should be different than the others. 🤔

Reply:
For sure, consistency would be good, and I'd be on board with opening issues to fix that, but in the mean time, we need to direct prometheus to scrape the port that's listening. The current config will just cause scrape errors.